server/target/target.go

```diff
@@ -12,13 +13,18 @@ import (
 	"github.com/buildbuddy-io/buildbuddy/server/util/query_builder"
 	"github.com/buildbuddy-io/buildbuddy/server/util/status"
 	"github.com/golang/protobuf/ptypes"
+	"google.golang.org/protobuf/proto"
 
 	cmpb "github.com/buildbuddy-io/buildbuddy/proto/api/v1/common"
 	trpb "github.com/buildbuddy-io/buildbuddy/proto/target"
+	tppb "github.com/buildbuddy-io/buildbuddy/proto/target_pagination"
+	gitutil "github.com/buildbuddy-io/buildbuddy/server/util/git"
 )
 
 const (
 	sqlite3Dialect = "sqlite3"
+	ciRole         = "CI"
+	targetPageSize = 20
```

Comment:
nit: can you comment what this field is? (assume some doofus like me comes along and needs to change it, do i have enough info in the comment to do that)

Reply:
Done